queue/priority_queue.go

```diff
@@ -120,8 +120,9 @@ func (pq *PriorityQueue) Put(items ...Item) error {
 	}
 
 	pq.lock.Lock()
+	defer pq.lock.Unlock()
```

Comment:
Good catch. I changed that part of the code just to be consistent with the rest of the functions.

Anyways, how can you test it consistently? I just ran two benchmarks, one was faster and one was slower than the current version:

```
workiva/go-datastructures > go test -bench .
PASS
BenchmarkPriorityQueue	 1000000	      2272 ns/op
BenchmarkQueue	 5000000	       388 ns/op
BenchmarkChannel	 1000000	      1129 ns/op
BenchmarkQueuePut	    5000	    243654 ns/op
BenchmarkQueueGet	   10000	    211298 ns/op
BenchmarkQueuePoll	   10000	    178354 ns/op
BenchmarkExecuteInParallel	   10000	    105245 ns/op
BenchmarkRBLifeCycle	  300000	      5260 ns/op
BenchmarkRBPut	10000000	       749 ns/op
BenchmarkRBGet	10000000	       193 ns/op
ok  	github.com/workiva/go-datastructures/queue	76.658s
```

```
jlhonora/go-datastructures > go test -bench .
PASS
BenchmarkPriorityQueue	 1000000	      2527 ns/op
BenchmarkQueue	 5000000	       386 ns/op
BenchmarkChannel	 1000000	      1147 ns/op
BenchmarkQueuePut	    5000	    247075 ns/op
BenchmarkQueueGet	   10000	    329593 ns/op
BenchmarkQueuePoll	   10000	    231860 ns/op
BenchmarkExecuteInParallel	   10000	    108442 ns/op
BenchmarkRBLifeCycle	  300000	      5350 ns/op
BenchmarkRBPut	 5000000	       230 ns/op
BenchmarkRBGet	10000000	       644 ns/op
ok  	github.com/jlhonora/go-datastructures/queue	59.819s
jlhonora/go-datastructures > go test -bench .
PASS
BenchmarkPriorityQueue	 1000000	      2350 ns/op
BenchmarkQueue	 5000000	       373 ns/op
BenchmarkChannel	 1000000	      1136 ns/op
BenchmarkQueuePut	    5000	    239267 ns/op
BenchmarkQueueGet	   10000	    191165 ns/op
BenchmarkQueuePoll	   10000	    181181 ns/op
BenchmarkExecuteInParallel	   10000	    152712 ns/op
BenchmarkRBLifeCycle	  300000	      7282 ns/op
BenchmarkRBPut	 5000000	       219 ns/op
BenchmarkRBGet	10000000	       511 ns/op
ok  	github.com/jlhonora/go-datastructures/queue	54.827s
```

Do you have a more standard way to run the benchmark suite?